.env.example

```diff
@@ -11,3 +11,5 @@ FEATURE_FOOTER_BUG_BUTTON=on
 FEATURE_SEND_FORM=
 MIXPANEL_TOKEN_PUBLIC=
 FEATURE_LOYALTY_FLOW=off
+GOOGLE_ANALYTICS_MEASUREMENT_ID=
+GOOGLE_ANALYTICS_API_SECRET=
```

Comment:
This api key will be exposed. Probably it is ok, but everything that has secret in its name raises a concern

Reply:
👍  Agree, it shouldn't cause much harm, but I'd prefer not to expose it

Reply:
Network traffic inspection could indeed reveal these GA API tokens... Could be solved by creating our custom endpoint, just like we did for access to firebase config. I'm just not sure it's worth it at this point